### PR TITLE
Local Process Kubernetes: add routing steps

### PR DESCRIPTION
**Pull request by ghogen**

### CODE REVIEW

**Comment by ghogen** [comment]
@GordonBy @greenie-msft 
This PR adds the routing steps to the VS Code LPK tutorial.
This is dependent upon a VS PR (https://github.com/MicrosoftDocs/visualstudio-docs-pr/pull/6881) that adds the routing section to the LPK Overview (since that is only hosted in the VS docs, not in VS Code).

File: docs/containers/local-process-kubernetes.md [hunk 1]
```diff
@@ -116,11 +116,15 @@ Choose **Launch via NPM** as the launch task.
...
+
+![Isolation prompt](images/local-process-kubernetes-vs-code/lpk-isolation-prompt.png)
+
 Your development computer is connected when the VS Code status bar turns orange and the Dev Spaces extension shows you are connected.
 
 ![Development computer connected](images/local-process-kubernetes-vs-code/development-computer-connected.png)
 
-> **Note**: On subsequent launches, you will not be prompted for the service name, port, or launch task. These values are saved in `.vscode/tasks.json`.
+> **Note**: On subsequent launches, you will not be prompted for the service name, port, launch task, or whether to run isolated. These values are saved in `.vscode/tasks.json`. To change these settings later, open the Command Palette, and run the command `Local Process Kubernetes: Configure Local Process Kubernetes`.
```
Comment (gregvanl) [inline docs/containers/local-process-kubernetes.md:127]: Commands which are displayed in UI like the Command Palette should be bold.
run the command **Local Process Kubernetes: Configure Local Process Kubernetes**.

**Comment by ghogen** [comment]
@Gordonby @greenie-msft 
Changes for 7/22 release

File: docs/containers/local-process-kubernetes.md [hunk 2]
```diff
@@ -29,8 +29,7 @@ This guide uses the [Bike Sharing sample application][bike-sharing-github] to de
 * An Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/free).
 * [Azure CLI installed][azure-cli].
 * [Visual Studio Code][vs-code] running on macOS or Windows 10.
-* The [Azure Dev Spaces][azds-vs-code] extension version 2.0.220200601 or greater installed in Visual Studio Code.
-* [Azure Dev Spaces CLI installed][azds-cli].
+* The [Local Process Kubernetes][lpk-vs-code] extension installed in Visual Studio Code.
```
Comment (Gordonby) [inline docs/containers/local-process-kubernetes.md:32]: "Local Process with Kubernetes"

**Comment by ghogen** [comment]
@greenie-msft Please review some final edits based on the live extension, and then we can publish.

**Comment by ghogen** [comment]
@gregvanl  This is ready to publis

**Pull request merged**